docs/install/installEverest.md

```diff
@@ -20,76 +20,57 @@ To install and provision Percona Everest to Kubernetes:
     ```sh
     everestctl install
     ```
-    
-    This will install all the required components in a namespace called `percona-everest`.
 
-    ??? example "Example"
-            
-            ? Namespace to deploy Everest to percona-everest
-            ? What operators do you want to install? MySQL, MongoDB, PostgreSQL
-            
+3. Enter the specific names for the namespaces you want Everest to manage, separating each name with a comma.
```

Comment:
I wouldn't set this as a completely separate step, we are basically just filling in the details of the command we run on step 2. I'd mention this as being part of step 2.

Reply:
Agreed! Generally, one task one step is being followed in procedure writing.